metricbeat/docs/fields.asciidoc

```diff
@@ -2590,7 +2590,7 @@ Contains statistics related to the coreDNS service
 
 
 
-*`coredns.stats.panic.count.total`*::
+*`coredns.stats.panic.count`*::
```

Comment:
you didn't mention this change, I understand it's intended?

Reply:
so true,

I'm following this convention here:
https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html#abbreviations

If you think this is ok, I'll go on and add it to the PR description

Reply:
SGTM

Reply:
First worried about the change as it's breaking but realised the CoreDNS module is not release yet. So all good.